apps/aeoracle/src/aeo_query.erl

```diff
@@ -8,7 +8,8 @@
 -module(aeo_query).
 
 %% API
--export([ deserialize/1
+-export([ add_response/3
```

Comment:
set_response/2 is now_unused and it shall be deleted? (alongside tests for this test).